examples/llava/qwen2vl-cli.cpp

```diff
@@ -367,14 +370,14 @@ static void debug_test_mrope_2d() {
     // 1. Initialize backend
     ggml_backend_t backend = NULL;
     std::string backend_name = "";
```

Comment:
Btw, I have https://github.com/ngxson/ggml-easy which may help in this case (I planned to support exporting tensor data as image)

Maybe worth exploring how to compile it with `ExternalProject`, so it can be an optional deps (without including the whole file into llama.cpp like httplib or json.hpp)